doc/source/parameters/dem/lagrangian_physical_properties.rst

```diff
@@ -86,12 +68,14 @@ In this subsection, gravitational acceleration, and the physical properties of t
 * The ``size distribution type`` parameter specifies the size distribution for each particle type. The acceptable choices are ``uniform`` and ``normal`` distributions.
 
 .. note::
-    For each particle type, two ``size distribution type``s can be defined: ``uniform`` and ``normal``. In ``uniform`` size distribution, the diameter of the particles is constant, while in ``normal`` size distribution, the particle diameters are sampled from a normal distribution with an average of ``average diameter`` and standard deviation of ``standard deviation``.
+    For each particle type, three ``size distribution type``s can be defined: ``uniform``, ``normal`` and ``histogram``. In ``uniform`` size distribution, the diameter of the particles is constant. In ``normal`` size distribution, the particle diameters are sampled from a normal distribution with an average diameter and a standard deviation of ``standard deviation``.  In the "histogram" size distribution, particle diameters are sampled from a list of diameters and a probability for each diameter value.
 
 * The ``diameter`` parameter defines the diameter of the particles in a ``uniform`` distribution. In the case of a ``normal`` distribution, this parameter indicates the average diameter.
 
 * For a ``normal`` distribution, the ``standard deviation`` parameter should be defined to indicate the standard deviation on the particle size distribution.
 
+* For a ``histogram`` distribution, the ``histogram diameters list`` defines each diameter to choose from when generating the diameter values. The ``histogram probabilities list`` defines the probability that a diameter value will be chosen for one particles. Both list must have the same length.
+
```

Comment:
Again, do you want mass fraction (volume fraction) or number fraction?
I would use a volume fraction as a specifier instead of a number fractoin